doc/v2/build_and_install/index_en.rst

```diff
@@ -1,32 +1,57 @@
-Install and Build
-=================
+install and compile
+==========
 
 .. _install_steps:
 
-Install Steps
-++++++++
+PaddlePaddle provides various ways of installation for many different users
 
-You can choose either pip or Docker to complete your install:
+focus on deep learning model development
+-----------------
+
+PaddlePaddle provides lots of packages of python wheel , that pip can install:
 
 .. toctree::
-   :maxdepth: 1
+	:maxdepth: 1
 
-   pip_install_en.rst
-   docker_install_en.rst
+	pip_install_cn.rst
 
-Build from Source
------------------
+this is the convenient way to install it , please choose the right installation package with mochine configure and system。
+
+follow the bottom frame
+----------
+
+PaddlePaddle provides the installation ways of Docker, please follow the tutorial:
+
+.. toctree::
+	:maxdepth: 1
+
+	docker_install_cn.rst
+
+we recommend running PaddlePaddle in docker , this way has more superiority ：
 
-..  warning::
+- don't need the third dependent of installing 
```

Comment:
don't need the third dependent of installing ->Does not require the installation of third-party dependencies.